kpops/utils/dict_differ.py

```diff
@@ -16,9 +16,9 @@ class DiffType(str, Enum):
     REMOVE = "remove"
 
 
-class Change(BaseModel):
-    old_value: Any | None
-    new_value: Any | None
```

Comment:
I am a bit puzzled why this was introduced. A type annotation of `Any` is worthless to us, as it hides any meaningful information we have on the value and makes it more error-prone and complex to narrow. Even `Unknown` is usually better because the type checker can infer the type.

Reply:
@raminqaf can explain. I think the return value of the diff package can have different types

Reply:
We should then change it to an `Uknown` I did in that way because the diff was returning various types back then